dashboard/src/modules/components/LoadingComponent/index.less

```diff
@@ -1,5 +1,6 @@
 .main-page-container {
     position: relative;
+    height: 100%;
```

Comment:
Just out of curiosity, what does this directive do?  (E.g., what is the default for `height` if you omit this line?)

Reply:
`main-page-container `is inside `PageSection` of `Page` Component.So, by giving height as 100% it takes the total height of the parent.

If we omit this, it's height will be the size of the content inside this container. 

If the content is absolutely positioned then it would overlap on the Header.

Reply:
> by giving height as 100% it takes the total height of the parent.

Does that make it likely that it will overlap with other things inside the parent which are outside this container?

And, if it's positioned relative to something else in the parent container (i.e., if it's not positioned at the top of the parent), then won't its height end up being too big for the parent container?  (I.e., if it's the same size as the parent but starts partway down, then some of it will extend beyond the end of the parent...right?)

Reply:
This is the main container, the only child of the parent container. All other components will be inside this container.
So nothing will go out of this parent.